pages/dao/[symbol]/proposal/components/instructions/UXDProtocol/EditController.tsx

```diff
@@ -0,0 +1,151 @@
+import * as yup from 'yup'
+
+import Switch from '@components/Switch'
+import { useContext, useEffect, useState } from 'react'
+import useGovernanceAssets from '@hooks/useGovernanceAssets'
+import {
+  Governance,
+  ProgramAccount,
+  serializeInstructionToBase64,
+} from '@solana/spl-governance'
+import useWalletStore from 'stores/useWalletStore'
+import { NewProposalContext } from '../../../new'
+import Input from '@components/inputs/Input'
+import {
+  UiInstruction,
+  UXDEditControllerForm,
+} from '@utils/uiTypes/proposalCreationTypes'
+import { isFormValid } from '@utils/formValidation'
+import { Controller, UXD_DECIMALS } from '@uxd-protocol/uxd-client'
+import { uxdClient } from '@tools/sdk/uxdProtocol/uxdClient'
+import GovernedAccountSelect from '../../GovernedAccountSelect'
```

Comment:
nit, import order (external dependencies first):

```suggestion
import * as yup from 'yup'
import { useContext, useEffect, useState } from 'react'
import { Controller, UXD_DECIMALS } from '@uxd-protocol/uxd-client'
import {
  Governance,
  ProgramAccount,
  serializeInstructionToBase64,
} from '@solana/spl-governance'
import Switch from '@components/Switch'
import useGovernanceAssets from '@hooks/useGovernanceAssets'
import useWalletStore from 'stores/useWalletStore'
import { NewProposalContext } from '../../../new'
import Input from '@components/inputs/Input'
import {
  UiInstruction,
  UXDEditControllerForm,
} from '@utils/uiTypes/proposalCreationTypes'
import { isFormValid } from '@utils/formValidation'
import { uxdClient } from '@tools/sdk/uxdProtocol/uxdClient'
import GovernedAccountSelect from '../../GovernedAccountSelect'
```